pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -325,6 +326,22 @@ func (w *Webhook) inject(pod *corev1.Pod, _ string, _ dynamic.Interface) error {
 	return w.injectAutoInstruConfig(pod, libsToInject, autoDetected, injectionType)
 }
 
+func injectSecurityClientLibraryProducts(pod *corev1.Pod) {
```

Comment:
nit: I think `injectSecurityClientLibraryConfig` would be clearer and more inline with the naming of other functions.

Reply:
Updated in 4406e81130